softdev.bib

```diff
@@ -2757,3 +2757,43 @@ @misc{watson21assessing
     month = Sep,
     url = "https://www.capabilitieslimited.co.uk/pdfs/20210917-capltd-cheri-desktop-report-version1-FINAL.pdf"
 }
+
+@misc{servo,
+    title = {The Servo Parallel Browser Engine Project},
+    howpublished = {\href{https://servo.org}{https://servo.org}},
+    note = {Accessed: 2022-12-17},
+    year = {2012},
+    author = {Mozilla},
+}
+
+@misc{rustgc,
+    title = {rust-gc: simple tracing (mark and sweep) garbage collector for Rust},
+    howpublished = {\href{https://github.com/manishearth/rust-gc/}{https://github.com/manishearth/rust-gc/}},
+    note = {Accessed: 2022-12-17},
+    year = {2015},
+    author = {Manish Goregaokar},
```

Comment:
This one I'm not too sure about. Manish owns the repo, but it looks like the bulk of contributions came from someone else.

Reply:
In such a case you don't even really need an author.

Reply:
Fixed: a123b4509127c3957bcc7e8d67f6e53d30ad9af4

Reply:
Argh! Actually this doesn't work, because prebib complains it's missing an author field.